server/src/domain/media/media.util.ts

```diff
@@ -615,7 +615,15 @@ export class RKMPPConfig extends BaseHWConfig {
     if (this.devices.length === 0) {
       throw new Error('No RKMPP device found');
     }
-    return [];
+    return ['-hwaccel rkmpp', '-hwaccel_output_format drm_prime', '-afbc rga'];
+  }
+
+  getFilterOptions(videoStream: VideoStreamInfo) {
+    const options = this.shouldToneMap(videoStream) ? this.getToneMapping() : [];
+    if (this.shouldScale(videoStream)) {
+      options.push(`scale_rkrga=${this.getScaling(videoStream)}:format=nv12:afbc=1`);
```

Comment:
Would it be faster to put this before the tone-mapping? I'm thinking it'd let the decoding and scaling happen together and leave a lower resolution video to tone-map.

Reply:
indeed. however, tonemapping does not work at all, so I've disabled it for now